dill/session.py

```diff
@@ -213,32 +305,39 @@ def dump_module(
 
     from .settings import settings
     protocol = settings['protocol']
+    if refimported is None:
+        refimported = settings['dump_module']['refimported']
```

Comment:
see my comment in `dill.settings` about a nested settings dict

Reply:
done